cmd/server/server.go

```diff
@@ -152,7 +151,26 @@ func (s *server) startService() common.Daemon {
 		s.cfg.Archival.DefaultBucket,
 		enableReadFromArchival(),
 	)
-	params.DispatcherProvider = client.NewIPYarpcDispatcherProvider()
+
+	// TODO this DispatcherProvider will also be used for clientBean to create dispatchers for remoteCluster in CrossDC re-replication.
```

Comment:
When adding `TODO` either point to an issue or point to `@<githubid>` otherwise these are sure to be orphaned.

Reply:
not applicable anymore